<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: #1e1e1e;
	color: #ffffff;
}

QPushButton {
	font: 63 20pt &quot;JetBrains Mono NL SemiBold&quot;;
	background-color: #9a9ef9;
	border: 1px solid #9a9ef9;
	border-radius: 20px;
	color: #FFFFFF;
}
QPushButton:hover {
    border-radius:20px;
	border: 1px solid #7070d0;
	background-color: #7070d0;
    color: white;
}

QPushButton:disabled {
    background-color: gray;
    color: white;
}

QPushButton:pressed {
    
	background-color: rgb(85, 0, 255);
    color: white;
}
QHeaderView::section {
 background-color: #303030;
border:none;
 }

QScrollBar {
        background-color: #333333;
        width: 10px;
    }
    QScrollBar::handle {
        background-color: #555555;
        min-height: 20px;
    }
    QScrollBar::handle:hover {
        background-color: #777777;
    }
    QScrollBar::add-line,
    QScrollBar::sub-line {
        background: none;
    }
    QScrollBar::add-page,
    QScrollBar::sub-page {
        background: none;
    }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
padding:1px</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1151</width>
        <height>518</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>25</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="go_back_button">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>data/left-arrow.png</normaloff>data/left-arrow.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_new_card_button">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
